projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/TestJobDeployTempCredsIT.java

```diff
@@ -98,6 +104,9 @@ public void setup() throws Exception {
                                 String.format(
                                     "/data-jobs/for-team/%s/jobs/%s",
                                     TEST_TEAM_NAME, TEST_JOB_NAME)))));
+    // Check authentication credentials are filled properly before continuing test.
+    Assertions.assertFalse(iamServiceAccountAccessKeyId.isBlank());
+    Assertions.assertFalse(iamUserServiceAccountSecretAccessKey.isBlank());
 
```

Comment:
nit: does it make sense to check on datajobs.aws.assumeIAMRole as well

Reply:
No because we explicitly set it at the start of the test.